stellar/deploy-contract.js

```diff
@@ -101,35 +99,25 @@ async function processDeploy(options, config, chain) {
     await broadcast(operation, wallet, chain, 'Initialized contract', options);
 }
 
-async function processUpgrade(options, _, chain) {
-    const { contractName, contractId, privateKey, wasmPath, newWasmHash } = options;
+async function upgrade(options, _, chain) {
+    const { privateKey, wasmPath } = options;
     const { rpc, networkType } = chain;
     const networkPassphrase = getNetworkPassphrase(networkType);
+    const contractAddress = chain.contracts?.axelar_gateway.address;
 
-    if (!chain.contracts) {
-        chain.contracts = {};
+    if (!contractAddress) {
```

Comment:
Look into using `validateParameters` (see existing usage) for strict and convenient validation of one or more params

Reply:
good call, updated.